components/common/Card/Card.tsx

```diff
@@ -1,54 +1,66 @@
 import { useState, MouseEvent } from "react";
+import Image from "next/image";
 import styles from "./card.module.css";
-import { formatDate, getTimeDifference } from "utils/helpers/dateUtils";
-import DeleteModal from "components/Modal/DeleteModal/DeleteModal";
-import FolderModal from "components/Modal/FolderModal/FolderModal";
-import noImage from "assets/images/bg_noImage.png";
-import meatballsIcon from "assets/images/ic_meatballs.svg";
-import starIcon from "assets/images/ic_star.svg";
-import { DELETE_LINK, ADD_LINK } from "utils/constants/strings";
-import { GetLinkResponse } from "types/apis";
-import { UseModal, Modal, OpenModal, CloseModal } from "utils/hooks/useModal";
+import DeleteModal from "@/components/common/Modal/DeleteModal/DeleteModal";
+import FolderModal from "@/components/common/Modal/FolderModal/FolderModal";
+import { formatDate, getTimeDifference } from "utils/dateUtils";
```

Comment:
utils라는 이름이 앞에 있기때문에 date로만 표현해줘도 충분할 것 같네요!
```suggestion
import { formatDate, getTimeDifference } from "utils/date";
```